packages/containers/psammead-timestamp-container/src/index.jsx

```diff
@@ -26,10 +25,6 @@ const TimestampContainer = ({
     return null;
   }
 
-  if (locale) {
-    moment.locale(locale);
-  }
-
   return (
     <Timestamp
       datetime={formatUnixTimestamp(timestamp, dateTimeFormat, timezone)}
```

Comment:
Your PR description says

>Remove moment.locale from the container as it is already in the formatUnixTimestamp function, to avoid conflicts.

But we arnt passing locale to `formatUnixTimestamp` here? Surely it defaults to `en-gb`?

Reply:
I believe so, beacuse we set it to `en-gb` in the [formatUnixTimestamp](https://github.com/bbc/psammead/blob/latest/packages/containers/psammead-timestamp-container/src/timestampUtilities.js#L38) function, if not locale is passed. 

@HarveyPeachey is actually planning to always pass the `en-gb`, instead of seting a default in [this](https://github.com/bbc/psammead/blob/latest/packages/containers/psammead-timestamp-container/src/timestampUtilities.js#L38) PR.

Reply:
I believe so, because we set it to `en-gb` in the [formatUnixTimestamp](https://github.com/bbc/psammead/blob/latest/packages/containers/psammead-timestamp-container/src/timestampUtilities.js#L38) function, if there is no local passed. 

@HarveyPeachey is actually planning to always pass the `en-gb`, instead of setting a default in [this](https://github.com/bbc/psammead/pull/2303/files#diff-9167111d8797dcd3c0e7444464a1d1c7R50) PR.